moduleroot/.github/workflows/ci.yml.erb

```diff
@@ -0,0 +1,106 @@
+name: CI
+
+on:
+  pull_request:
+  schedule:
+    - cron: '4 4 * * *'
```

Comment:
I wish we could have some kind of splay here so that not *all* modules get scheduled at the same time.

Sadly, GHA does not support that out of the box (https://docs.github.com/en/actions/reference/events-that-trigger-workflows#schedule)

Reply:
This *is* ERB, we could generate a random number on every generation of this to ensure some amount of splay? That would in theory mean every repo would end up potentially with a different entry.

Reply:
We do have some ip2cron logic in puppet-puppet which essentially takes an IP as a seed. We can take the repo name and generate a random number based on that. On the other hand, the time slot is in the middle of the night and Github can handle the queuing for us. Not sure how much we would gain from it.